daisy_workflows/image_import/inspection/README.md

```diff
@@ -57,13 +57,14 @@ boot-inspect /images/ubuntu-16.04-server-cloudimg-amd64-disk1.vmdk
 The safe and quick option is to run the prow job's container locally:
 
 ```shell script
+cd $COMPUTE_IMAGE_TOOLS
```

Comment:
Is $COMPUTE_IMAGE_TOOLS defined?

Reply:
The example assumed that variable pointed to the root of the compute-image-tools repo. Clearly that was a bad assumption :)  I removed the variable, and wrote a comment to run the example from the root of the repo.